signer/src/proto/convert.rs

```diff
@@ -404,6 +1701,8 @@ mod tests {
     #[test_case(PhantomData::<(SignerWithdrawalDecision, proto::SignerWithdrawalDecision)>; "SignerWithdrawalDecision")]
     #[test_case(PhantomData::<(BitcoinTransactionSignAck, proto::BitcoinTransactionSignAck)>; "BitcoinTransactionSignAck")]
     #[test_case(PhantomData::<(StacksTransactionSignature, proto::StacksTransactionSignature)>; "StacksTransactionSignature")]
+    #[test_case(PhantomData::<(SignerMessage, proto::SignerMessage)>; "SignerMessage")]
+    #[test_case(PhantomData::<(Signed<SignerMessage>, proto::Signed)>; "Signed")]
```

Comment:
My concern here was that maybe we're not ensuring that we're testing all of the possible `Payload` combinations, and thus the code above, since it will be faked (randomized). Not sure if there's other tests somewhere covering this though.

Reply:
Yeah, I have the same concern. We should add test cases for the types individually.

Reply:
Added the tests placeholder in here, I'll enable them (with the required mocking) in a separate PR (before merging https://github.com/stacks-network/sbtc/pull/880)

Reply:
Done in https://github.com/stacks-network/sbtc/pull/899